packages/common/src/store/purchase-content/sagas.ts

```diff
@@ -232,6 +233,10 @@ function* doStartPurchaseContentFlow({
       yield* put(saveTrack(contentId, FavoriteSource.IMPLICIT))
     }
 
+    // clear the purchases so next query will fetch from source
+    // @ts-ignore
```

Comment:
getting `Type 'void' has no call signatures.` without the ts-ignore. trying to remove this error without the ts-ignore